governance/votes/Executive vote - September 24, 2021.md

```diff
@@ -0,0 +1,73 @@
+date: 2021-09-24T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Adjusting Auction Parameters and Testing Layer 2 Components - September 24, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The following adjustments to Auction Parameters will be made:
+  - **Decrease** the [Auction Price Multiplier (buf)](https://makerdao.world/en/learn/governance/param-auction-price-multiplier) for ETH-A, ETH-C, and WBTC-A vaults from 1.3 to **1.2**
+  - **Increase** the ETH-A [Local Liquidation Limit (ilk.hole)](https://makerdao.world/en/learn/governance/param-local-liquidation-limit) from 30 million DAI to **40 million DAI**
+  - **Increase** the ETH-B [Local Liquidation Limit (ilk.hole)](https://makerdao.world/en/learn/governance/param-local-liquidation-limit) from 15 million DAI to **25 million DAI**
+  - **Increase** the ETH-C [Local Liquidation Limit (ilk.hole)](https://makerdao.world/en/learn/governance/param-local-liquidation-limit) from 20 million DAI to **30 million DAI**
+  - **Increase** the WBTC-A [Local Liquidation Limit (ilk.hole)](https://makerdao.world/en/learn/governance/param-local-liquidation-limit) from 15 million DAI to **25 million DAI**
+- Multiple **test actions** will take place for Layer 2 DAI solutions on Optimism and Arbitrum, with the intention of denying the Protocol Engineering Core Unit's Deployer Keys if successful. 
+
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
```

Comment:
We have office hours for this? If so, we should maybe remove the reason here, as there are no new collateral types.

Reply:
✅